telemetry/definitions/commonDefinitions.json

```
@@ -2177,6 +2182,24 @@
             ],
             "passive": true
         },
+        {
+            "name": "toolkit_showNotification",
+            "description": "The toolkit tried to retrieve and show notification message",
```

looks like "retrieve" is left over from copy-paste from `toolkit_getExternalResource` ? :) 

```suggestion
            "description": "The toolkit tried to show a notification message",
```